Lib/pdb.py

```diff
@@ -1890,11 +1936,16 @@ def set_trace(*, header=None):
 # Post-Mortem interface
 
 def post_mortem(t=None):
-    """Enter post-mortem debugging of the given *traceback* object.
+    """Enter post-mortem debugging of the given *traceback*, or *exception*
+    object.
 
     If no traceback is given, it uses the one of the exception that is
     currently being handled (an exception must be being handled if the
     default is to be used).
+
+    If `t` is an Exception and is a chained exception (i.e it has a __context__),
+    pdb will be able to move to the sub-exception when reaching the bottom
```

Comment:
```suggestion
    pdb will be able to move to other exceptions in the chain when reaching the bottom
```

(sub-exception sounds like it belongs to exception groups, and you're not dealing with those here).

Reply:
That said, I don't think it's clear from the context what "when reaching the bottom" means.

Reply:
Sorry that's a leftover when up/down were moving to previous/next exception in the chain when reaching the bottom/top of the stack, but we removed in in favor of do_exception. rephrased.